hazelcast-hibernate53/src/main/java/com/hazelcast/hibernate/local/LocalRegionCache.java

```diff
@@ -91,27 +96,29 @@ public LocalRegionCache(final RegionFactory regionFactory, final String name,
-     *                          configuration and to lookup an {@link ITopic} to register a {@link MessageListener}
-     *                          with if {@code withTopic} is {@code true} (optional)
-     * @param regionConfig      the region configuration
-     * @param withTopic         {@code true} to register a {@link MessageListener} with the {@link ITopic} whose name
-     *                          matches this region cache <i>if</i> a {@code HazelcastInstance} was provided to look
-     *                          up the topic; otherwise, {@code false} not to register a listener even if an instance
-     *                          was provided
-     * @param evictionConfig    provides the parameters which should be used when evicting entries from the cache;
-     *                          if null, this will be derived from the Hazelcast {@link MapConfig}; if the MapConfig
-     *                          cannot be resolved, this will use defaults.
+     * @param regionFactory             the region factory
+     * @param name                      the name for this region cache, which is also used to retrieve configuration/topic
+     * @param hazelcastInstance         the {@code HazelcastInstance} to which this region cache belongs, used to retrieve
+     *                                  configuration and to lookup an {@link ITopic} to register a {@link MessageListener}
+     *                                  with if {@code withTopic} is {@code true} (optional)
+     * @param regionConfig              the region configuration
+     * @param withTopic                 {@code true} to register a {@link MessageListener} with the {@link ITopic} whose name
+     *                                  matches this region cache <i>if</i> a {@code HazelcastInstance} was provided to look
+     *                                  up the topic; otherwise, {@code false} not to register a listener even if an instance
+     *                                  was provided
+     * @param evictionConfig            provides the parameters which should be used when evicting entries from the cache;
+     *                                  if null, this will be derived from the Hazelcast {@link MapConfig}; if the MapConfig
+     *                                  cannot be resolved, this will use defaults.
+     * @param freeHeapBasedCacheEvictor performs the free-heap-based eviction
      */
     public LocalRegionCache(final RegionFactory regionFactory, final String name,
                             final HazelcastInstance hazelcastInstance, final DomainDataRegionConfig regionConfig,
-                            final boolean withTopic, final EvictionConfig evictionConfig) {
+                            final boolean withTopic, final EvictionConfig evictionConfig,
+                            FreeHeapBasedCacheEvictor freeHeapBasedCacheEvictor) {
```

Comment:
```suggestion
                            final FreeHeapBasedCacheEvictor freeHeapBasedCacheEvictor) {
```
Just for the sake of consistency.

Reply:
fixed: https://github.com/hazelcast/hazelcast-hibernate/pull/355/commits/9bb7ee2756cc25093add34c19530e2ba7c90d98a